src/app/my-account/my-account/my-account.component.ts

```diff
@@ -44,7 +53,34 @@ export class MyAccountComponent implements OnInit {
     dialogRef.componentInstance.onClosed.pipe(take(1)).subscribe((confirmed: boolean) => {
       dialogRef.close();
       if (confirmed) {
-        // TODO: call to api
+        if (announcement.type === TransportOffer.TypeEnum.Transport) {
+          this.transportResourceService
+            .deleteTransport(announcement.id!)
+            .pipe(
+              switchMap((_) => {
```

Comment:
If we don't use the parameter then let's not pass it, why introduce an additional variable to the scope.

The same below in a few places.